Sources/muterCore/TestReporting/HTML/BundleResource.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension Bundle {
+    static func resource(named: String, ofType type: String) -> String {
+        Bundle.module.path(forResource: named, ofType: type)
+            .flatMap {
+                try? String(contentsOfFile: $0)
```

Comment:
Do we want this to fail silently?

Reply:
I think so...the html report won't look as good, but at least we are not going to waste user's time!